x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts

```diff
@@ -30,122 +31,90 @@ import { FtrProviderContext } from '../../../../../ftr_provider_context';
 import { getWebHookConnectorParams } from '../../../utils/connectors/get_web_hook_connector_params';
 import { createConnector } from '../../../utils/connectors';
 
+const getBaseRuleWithActions = (
```

Comment:
nit: This function should return `RuleImport` type. Current return value is unbound. It's not bad in a general case when we wanna test forwards and backwards compatibility but it should be covered by specific tests.

Default actions looks unnecessary since two usages in `getImportRuleBuffer()` and in `getImportRuleWithConnectorsBuffer()` provide actions.

Should the name be changed to something much more verbose like `createRuleImportMock`?